### ISSUE TITLE
first run of unit tests optimization

### ISSUE THREAD

**Issue by Announcement** [firasdib/Regex101#977]
see here the first test is >300ms
but the other tests are 0.

[test unit delay](https://regex101.com/r/f9uNym/4)

probably caching or compiling or something just thought i'd point it out.

**Comment by firasdib** [comment]
Yeah, this is typically because it has to compile and cache the regex the first time its being ran. Not sure its worth working around this, but it might be possible to pre-compile the regex.
